Rendering/Tools/SplineInterpretation.h

```diff
@@ -121,6 +124,14 @@ namespace OpenInfraPlatform
 			 */
 			std::vector<double> movingAverageVariableWindow(std::vector<double>& data) const throw(...);
 
+			/* \brief In a vector of pairs, this function obtains the second pair value and saves the values into a new vector
+			 *
+			 * For example the curvature in the vector \c lengthsWithCurvatures can be obtained for further use.
+			 *
+			 * \param[in]	lengthsWithCurvatures	A vecotr of \c pair<double,double>.
+			 *
+			 * \return		A vector of \c double wiht the second values of the pair vector.
+			 */
 			std::vector<double> obtainCurvatureFrom_lengthWithCurvatures(const std::vector<std::pair<double, double>>& lengthsWithCurvatures) const throw(...);

```

Comment:
This is easily obtained with a `std::transform` locally.

Reply:
The function calls are replaced to `std::tansform` in commit 75dcd42. As well, one function overload of obtainThreshold is removed entirely, instead `std::transform` is used at its previous call.